### PR TITLE
Move all a ops to ov

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Depends on PR https://github.com/openvinotoolkit/openvino/pull/7274
 - All operations from A symbol moved to ov namespace
 - Constant, ReadValue and Sink also were moved

### CODE REVIEW

File: ngraph/core/src/op/acos.cpp [hunk 1]
```diff
@@ -18,30 +18,27 @@
 #include "ngraph/runtime/host_tensor.hpp"
 #include "ngraph/runtime/reference/acos.hpp"
 
-using namespace std;
-using namespace ngraph;
+NGRAPH_RTTI_DEFINITION(ov::op::v0::Acos, "Acos", 0);
```
Comment (mitruska) [inline ngraph/core/src/op/acos.cpp:21]:
```suggestion
OPENVINO_RTTI_DEFINITION(ov::op::v0::Acos, "Acos", 0, util::UnaryElementwiseArithmetic);
```

File: ngraph/core/src/op/abs.cpp [hunk 2]
```diff
@@ -10,30 +10,27 @@
 #include "ngraph/runtime/host_tensor.hpp"
 #include "ngraph/runtime/reference/abs.hpp"
 
-using namespace std;
-using namespace ngraph;
+NGRAPH_RTTI_DEFINITION(ov::op::v0::Abs, "Abs", 0);
```
Comment (mitruska) [inline ngraph/core/src/op/abs.cpp:13]:
```suggestion
OPENVINO_RTTI_DEFINITION(ov::op::v0::Abs, "Abs", 0, util::UnaryElementwiseArithmetic);
```

**Review by ilya-lavrenov — APPROVED** [review]

File: ngraph/core/src/op/acos.cpp (hunk 1 above)
Comment (ilyachur) [reply ngraph/core/src/op/acos.cpp:21]: I will fix it in the separate PR.

File: ngraph/core/src/op/abs.cpp (hunk 2 above)
Comment (ilyachur) [reply ngraph/core/src/op/abs.cpp:13]: Thanks, I will fix it in the separate PR.

**Pull request merged**